src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -7,6 +7,7 @@
     <TargetFrameworkNetCore>netcoreapp2.1</TargetFrameworkNetCore>
     <TargetFrameworkNet5Win>net5.0-windows10.0.17763.0</TargetFrameworkNet5Win>
```

Comment:
Can this also be changed to net6.0-windows10.0.19041.0?

Reply:
Not right now because net5-winX apps will then break. See #3700 for constraints. Also it's better to use lowest needed version - 17763 in this case.